subsys/bluetooth/gatt_dm.c

```diff
@@ -352,6 +363,7 @@ static uint8_t discovery_process_attribute(struct bt_gatt_dm *dm,
 
 	if (bt_uuid_cmp(attr->uuid, BT_UUID_GATT_CHRC) == 0) {
 		cur_attr = attr_store(dm, attr, sizeof(struct bt_gatt_chrc));
```

Comment:
We will still have invalid `bt_gatt_chrc::uuid` pointer. With this implementation we will have NULL, previously it was undefined.

Reply:
Modified. To avoid null pointers, the character's uuid pointer has the attribute's uuid value.